src/http/interceptor.js

```diff
@@ -0,0 +1,52 @@
+import axios from "axios";
+
+const webApi = axios.create({
+    baseURL: 'https://api.wisey.app/api/v1/core',
+});
+
+webApi.interceptors.request.use(async config => {
+    let token = '';
+
+    try {
+        const response = await axios.get('https://api.wisey.app/api/v1/auth/anonymous?platform=subscriptions');
+        token = response.data.token;
+    } catch (error) {
+        console.log(error);
+    }
+
+    config.headers.Authorization = `Bearer ${token}`;
```

Comment:
can you explain, please, why does function's argument `config` changes?
I've red somewhere, that changing arguments it is not a good solution, but probably you better understand the context

Reply:
If I'm not mistaken, here's set authorization header for all request, it's done only once. 

I'll learn this topic to search if it's a good solution. It's important and interesting, thanks!